2-mandatory/3-financial-times.js

```diff
@@ -24,6 +42,14 @@ function titleWithFewestWords(allArticleTitles) {
 */
 function headlinesWithNumbers(allArticleTitles) {
     // TODO
+    { let newClicksArray = []; for (let i = 0; i < allArticleTitles.length; i++) { 
+        for (let letter of allArticleTitles[i]) { 
+            if (letter === "0"|| letter === "1"|| letter === "2"|| letter === "3"|| letter === "4"|| letter === "5"|| letter === "6"|| letter === "7"|| letter === "8"|| letter === "9") 
+            { newClicksArray.push(allArticleTitles[i]) 
+        } 
+      }
+    } 
+    return newClicksArray; }
 }
 
 /*
```

Comment:
Have a look at this approach for this one: https://stackoverflow.com/questions/5778020/check-whether-an-input-string-contains-a-number-in-javascript